packages/tocco-ui-showcase/src/components/ShowCaseApp.spec.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
What's that needed for? If I remove this line and run the linter locally, it doesn't report any errors.